lib/ecto/multi.ex

```diff
@@ -8,7 +8,9 @@ defmodule Ecto.Multi do
   is given a name that is unique and will identify its result in case of
   success or failure.
 
-  All operations will be executed in the order they were added.
+  All operations will be executed in the order they were added, with the
+  possible exception of a multi which contains operations that accept
+  [changesets](#module-changesets).
```

Comment:
```suggestion
  If a multi is valid (i.e. all the changesets in it are valid), all operations
  will be executed in the order they were added.
```

WDYT?

Reply:
Definitely the more direct approach of explaining it. 👍